ads/adblade.md

```diff
@@ -21,8 +21,8 @@ limitations under the License.
 ```html
 <amp-ad width=300 height=250
     type="adblade"
-    width="300"
-    height="250"
+    data-width="300"
+    data-height="250"
```

Comment:
I was suggesting write them like this. adding the extra ` makes it look better. But it's fine, i am not that strong with the style
```
`data-height`
```